.github/workflows/main.yml

```diff
@@ -103,6 +103,22 @@ jobs:
         run: yarn --silent
       - name: run tests
         run: yarn test:lighthouse
+  pingdom:
+    strategy:
+      matrix:
+        platform: [ubuntu-latest]
+        node: [ '12' ]
+    name: pingdom - Node ${{ matrix.node }} (${{ matrix.platform }})
+    runs-on: ${{ matrix.platform }}
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node }}
+      - name: install dependencies
+        run: yarn --silent
```

Comment:
Nice idea! But can one bind node_modules dir as a volume in gh actions? Or is there anther way I'm not yet seeing, apart from combining all tasks in a single one which wouldn't work obviously, because we can't distinguish between failed modules.

Reply:
Maybe this can be done with dependent jobs: https://docs.github.com/en/actions/learn-github-actions/managing-complex-workflows#creating-dependent-jobs

Where the dependent job would restore cache + install dependencies + cache node_modules and all other jobs would use the same cache key and restore from it (without npm install).

Reply:
This is a separate task that I laid out https://github.com/website-checks/website-checks/issues/83